samples/dashboards/demo/sync-extremes/demo.js

```diff
@@ -97,10 +162,184 @@ Dashboards.board('container', {
             'Metro Area(km2)': null,
             'Highest Elevation(m)': 'y'
         },
-        chartOptions
+        chartOptions: {
+            xAxis: {
+                type: 'category',
+                labels: {
+                    enabled: false
+                }
+            },
+            yAxis: {
+                title: {
+                    text: ''
+                }
+            },
+            credits: {
+                enabled: false
+            },
+            chart: {
+                type: 'column',
+                zoomType: 'x'
+            },
+            plotOptions: {
+                series: {
+                    colorByPoint: true
+                },
+                tooltip: {
+                    headerFormat: '{point.key}',
+                    format: '{y}'
+                }
+            },
+            title: {
+                text: ''
+            },
+            subtitle: {
+                text: 'Meters',
+                align: 'left',
+                y: 0
+            },
+            legend: {
+                enabled: false
+            }
+        }
     },
     {
+        title: {
+            text: ''
+        },
         cell: 'dashboard-col-3',
+        type: 'Highcharts',
+        chartOptions: {
+            xAxis: {
+                visible: false
+            },
+            yAxis: {
+                title: {
+                    text: ''
+                },
+                height: 0
+            },
+            credits: {
+                enabled: false
+            },
+            chart: {
+                type: 'scatter',
+                margin: 0,
+                spacing: 0
+            },
+            plotOptions: {
+                series: {
+                    colorByPoint: true,
+                    marker: {
+                        symbol: 'square'
+                    },
+                    events: {
+                        legendItemClick: function () {
+                            return false;
+                        }
+                    }
+                }
+            },
+            title: {
+                text: ''
+            },
+            legend: {
+                title: {
+                    text: 'Cities'
+                },
+                enabled: true,
+                padding: 0,
+                floating: true,
+                y: -30,
+                navigation: {
+                    enabled: false
+                }
+            },
+            series: [{
```

Comment:
We need to remove all `series` options - it should be possible to get the same effect without `series`. 

Something like this: https://jsfiddle.net/BlackLabel/9zd4aLs1/3/ (in JS tab: modified config for the component, in CSS disabled inactive state at the bottom)